config/config.go

```diff
@@ -35,105 +37,134 @@ const (
 	OracleDisableCerts  = "disable_certs"
 )
 
-var defaultRootConfigContents = map[string]string{CurrentContext: "default"}
-var defaultContextConfigContents = map[string]string{
-	ContextProvider: "default",
-	EnvFnAPIURL:     "http://localhost:8080/v1",
+var DefaultRootConfigContents = map[string]string{CurrentContext: ""}
+var DefaultContextConfigContents = map[string]string{
+	ContextProvider: DefaultProvider,
+	EnvFnAPIURL:     DefaultLocalApiUrl,
 	EnvFnRegistry:   "",
 }
 
+type ContextFile struct {
+	ContextProvider string `yaml:"provider"`
+	EnvFnAPIURL     string `yaml:"api_url"`
+	EnvFnRegistry   string `yaml:"registry"`
+}
+
 // EnsureConfiguration ensures context configuration directory hierarchy is in place, if not
 // creates it and the default context configuration files
 func EnsureConfiguration() {
-	home, err := homedir.Dir()
+	home, err := GetHomeDir()
 	if err != nil {
-		panic(err)
+		fmt.Printf("%v", err)
```

Comment:
is this supposed to print to stderr?

Reply:
Oh yes thank you i missed one!

Reply:
@denismakogon does `log.Println`/`log.Fatal` go to stdout or stderr?

Reply:
Yes but fatal panics.